project/plugins.sbt

```diff
@@ -6,6 +6,6 @@ addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.5")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 
-addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
```

Comment:
updated version for upgrading to 2.13, but haven't run scalafmt yet...I'll run it once PR is either approved or merged

Reply:
I don't personally use scalafmt. I think I'd slightly prefer to see it as a different PR just to see what it looks like independent of the 2.13 upgrade, but I don't feel strongly about that by any means